fluentd/run.sh

```diff
@@ -1,280 +0,0 @@
-#!/bin/bash
```

Comment:
given we still have this as the Dockerfile `CMD` it is probably a good practice to keep this file, but we can update it to be more of a `usage` file in that it simply echos out the usage for this (saying it should be used in concert with CLO and a LogForwarding CR).

Reply:
restored run.sh, truncated the content